pkg/blobstore/configuration/create_blob_access.go

```diff
@@ -200,22 +200,46 @@ func createBlobAccess(configuration *pb.BlobAccessConfiguration, options *blobAc
 
 		tlsConfig, err := util.NewTLSConfigFromClientConfiguration(backend.Redis.Tls)
 		if err != nil {
-			return nil, err
+			return nil, util.StatusWrap(err, "failed to obtain new tls")
```

Comment:
The current convention is to capitalize all error messages, and to make abbreviations uppercase where applicable:
```go
return nil, util.StatusWrap(err, "Failed to obtain TLS configuration")
```